examples/20151026_SantaMonica/run_examples.py

```diff
@@ -1,25 +0,0 @@
-# David R Thompson, Adam Erickson
```

Comment:
Moved to `isofit/test/test_examples.py`. I don't see a reason to maintain multiple versions of the configuration.